src/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  padding: 0;
+}
+
+.list {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
what is list? it's separate BEM block? if yes - these styles should be in separate css file
but check, if you really need these styles